scipy/stats/tests/test_stats.py

```diff
@@ -5995,16 +6010,15 @@ def test_binomtest2():
          1.000000000, 0.753906250, 0.343750000, 0.109375000, 0.021484375,
          0.001953125]
     ]
-
     for k in range(1, 11):
-        res1 = [stats.binom_test(v, k, 0.5) for v in range(k + 1)]
+        res1 = [stats.binomtest(v, k, 0.5).pvalue for v in range(k + 1)]
```

Comment:
Is the addition of `.pvalue` here necessary? I would have expected `binomtest` and `binom_test` to have 100% compatible API.

Reply:
acc. to docs, it is. I also got an error w/o `.pvalue`

Reply:
In general we want to move away from the [named]tuple output for statistical tests. We see now that it limits us to add new outputs, etc.

Reply:
Agreed. But I think it's ok here since the function was already deprecated and here we are giving again 2 versions to adjust. And the change is fairly small.